app/src/main/java/org/simple/clinic/bp/history/adapter/BloodPressureHistoryListItem.kt

```diff
@@ -44,17 +44,13 @@ sealed class BloodPressureHistoryListItem : PagingItemAdapter.Item<Event> {
           .append(bloodPressureDateTime(context))
           .popSpan()
           .build()
-      val labelTextRes = measurement.level.displayTextRes.toNullable()
 
       if (isBpHigh) {
         holder.heartImageView.setImageResource(R.drawable.bp_reading_high)
       } else {
         holder.heartImageView.setImageResource(R.drawable.bp_reading_normal)
       }
       holder.bpHighTextView.visibleOrGone(isBpHigh)
```

Comment:
I am assuming that the value set in the layout for this label is "High", correct?

Reply:
Yep, I only used the `displayTextRes` in order to show the "Low" label text directly instead of checking it in the views.